### ISSUE TITLE
Projection - Select property of linked vertex and all '*'

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#5023]
OrientDB: 2.0.14
OrientJs

Works:

```
select *, SponsoredBy.Email as SponsorEmail from Node where Email = '<email>'
```

and event this works

```
select *, SponsoredBy.* as Sponsor from Node where Email = '<email>'
```

but

```
select SponsoredBy.Email as SponsorEmail, * from Node where Email = '<email>'
```

the field SponsorEmail is missing.

Look at **(*)**

**Comment by luigidellaquila** [comment]
Hi @StarpTech 

It's already fixed in 2.1, could you please check if it works in your case?

**Comment by StarpTech** [comment]
@luigidellaquila thanks. Gets the 2.0.X series also this fix?

**Comment by luigidellaquila** [comment]
Hi @StarpTech 
it was fixed with the new parser, so I'm afraid it won't be easy to port it to 2.0.x

**Comment by StarpTech** [comment]
Hi @luigidellaquila 
this would mean that everybody should use the new 2.1 releases?

**Comment by luigidellaquila** [comment]
Hi @StarpTech 

we strongly recommend it. Migration should be quite easy

**Comment by luigidellaquila** [comment]
Of course 2.0.x is still supported, so issues on that release can also go in high priority if customers ask.

**Comment by lvca** [comment]
Guys, is this already available on 2.1, right? Can I close the issue?

**Comment by prjhub** [comment]
The issue has been closed because no response has been received after a while. In case the issue is still valid, please reopen it or write a comment that explains why it's still valid and a way to reproduce it. Thank you.
